uportal-war/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp

```diff
@@ -35,10 +35,12 @@
         <c:set var="searchLabel"><spring:message code="search"/></c:set>
         <form class="form-search" role="form" method="post" action="${searchLaunchUrl}" id="${n}webSearchForm">
           <div class="input-group">
-            <input id="${n}webSearchInput"  class="searchInput input-large search-query form-control" value="" name="query" type="text" placeholder="Enter search terms"/>
+            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query" type="search" placeholder="Enter search terms" title="Search Terms"/>
```

Comment:
@ChristianMurphy Beware you've hardcoded the placeholder and the title strings in english. :-) 
Can you use message tags to localize those strings. The funny is that it will be also an accessibility issue in french if you don't change that (you need to indicate when the display switch from french (before the input) to english (the hardcoded placeholder... :-) 
I've checked the support of type="search" on caniuse -> 👍 nice done